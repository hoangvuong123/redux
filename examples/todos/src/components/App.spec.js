import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

const setup = (setupProps = {}) => {
  const wrapper = shallow(<App/>);
  return { wrapper };
};

describe('App component', () => {
  it('render App component', () => {
    const { wrapper } = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
