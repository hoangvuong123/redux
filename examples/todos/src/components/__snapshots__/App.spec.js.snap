// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render App component 1`] = `
<div>
  <Connect(AddTodo) />
  <Connect(TodoList) />
  <Footer />
</div>
`;
